### ISSUE TITLE
Firefox trouble with "modify team" feature

### ISSUE THREAD

**Issue by user23498723452** [hook/champions#14]
It seems to be broken.  No Apply button ever appears.  You can highlight characters up top and below, but just no apply button,  I think.

**Comment by hook** [comment]
It works for me in Firefox, IE, Chrome, Safari and Opera. Are you sure the Apply box doesn't show up just on the one you aren't expecting?

**Comment by user23498723452** [comment]
Hmm,  it seems to be working now.  But I did see one oddity where the button disappeared when something was selected.  Maybe there is a bug when you do some weird hero selection order (like top, top, bottom, top or something).  But I was consistently unable to get the Apply button to appear previously. 

I do use plugins in FF, but they do not appear to be the root cause as I have now tested with them enabled and disabled.  It "works" both ways . So I have to suspect some type of bug, possibly due to odd user decisions.
